contrib/nvmf_proxy-default

```diff
@@ -3,7 +3,7 @@
 # contains environment variables for /etc/systemd/system/nvmf_tgt.service
 
 # run on 4 cores out of 8 on SNIC
-CPU_MASK=0xf0
+CPU_MASK=0xaa
```

Comment:
In fact on 4 cores SNIC we have only cores 0-3, i.e. allowed mask is `0x0F`, not `0xAA`

Reply:
Like you suggested, it is bad idea to use all 4 cores, I preferred using only 2 cores.
So 0xAA is being "cut off" to just 0x0A (only 4 LSBs), and only 2 cores is allowed.